scope/fritz.py

```diff
@@ -293,15 +284,38 @@ def save_newsource(
                 "group_ids": group_ids,
                 "origin": "Fritz",
             }
-            response = api("POST", "/api/sources", token, post_source_data)
-            if response.json()["status"] == "error":
-                print(f"Failed to save {obj_id} as a Source")
-                return None
-
-        print("Found %d lightcurves" % len(light_curves))
-
-        # post photometry to obj_id; drop flagged data
-        df_photometry = make_photometry(light_curves, drop_flagged=True)
+            for attempt in range(10):
```

Comment:
Maybe make the number of attempts a variable like 
MAX_ATTEMPTS = 10
Defined at the top of the script?

Reply:
Agreed! I added this variable for both scripts and replaced all range(10) with range(MAX_ATTEMPTS).